packages/react-native-sdk/package.json

```diff
@@ -29,43 +29,42 @@
     "test": "jest"
   },
   "dependencies": {
-    "axios": "0.19.0",
-    "kinvey-js-sdk": "^5.1.1",
-    "lodash": "^4.17.15",
-    "pubnub": "4.25.2"
+    "axios": "0.21.1",
+    "kinvey-js-sdk": "6.0.0-next.1",
```

Comment:
Is this set temporary for running the tests, before we release the next major?

Reply:
Yes, exactly. We are going to release `6.0.0` core SDK with MFA and some other changes first, and then RN as a following version.